src/sections/pools/sections/IsolatedPools.tsx

```diff
@@ -125,7 +135,16 @@ const IsolatedPoolsData = () => {
       {xykPools.isInitialLoading ? (
         <PoolsTableSkeleton isXyk />
       ) : filteredPools.length ? (
-        <PoolsTable data={filteredPools} isXyk />
+        <PoolsTable
+          data={
+            showAllXyk
+              ? filteredPools
+              : filteredPools.filter((pool) =>
+                  pool.tvlDisplay.gte(XYK_TVL_VISIBILITY),
+                )
+          }
```

Comment:
maybe it would be better to put the filtering in useMemo, so it doesn't run filter on every re-render